Products/ZenModel/migrate/UpdateEditUserPermission.py

```diff
@@ -0,0 +1,52 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2018, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+
+__doc__ = '''
+This migration script puts back ZEN_MANAGER role to the ZEN_EDIT_USER permission.
+'''
+
+
+import Migrate
+
+from Products.ZenModel.ZenossSecurity import (
+    ZEN_MANAGER_ROLE,
+    MANAGER_ROLE,
+    CZ_ADMIN_ROLE,
+    OWNER_ROLE,
+    ZEN_EDIT_USER
+)
+
+
+class UpdateEditUserPermission(Migrate.Step):
+    
+    version = Migrate.Version(300, 0, 0)
```

Comment:
I'm thinking the migrate version should be higher than the version given in #3135.
The ordering of migrate scripts having the same version is undefined.